CMake AMReX version error

Hello, I am getting this error recently:
```
CMake Error at Submodules/AMReX/CMakeLists.txt:46 (project):
  VERSION "v0.1.146" format invalid.
```
I think it is due to https://github.com/AMReX-Codes/amrex/commit/15c6c1f8a9fa10630bac0089a994bf8602aa9f03

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -9,13 +9,20 @@ find_package (Git QUIET)
 
 set( _tmp "" )
 
+# Try to inquire software version from git
 if ( EXISTS ${CMAKE_CURRENT_LIST_DIR}/.git AND ${GIT_FOUND} )
    execute_process ( COMMAND git describe --abbrev=12 --dirty --always --tags
       WORKING_DIRECTORY ${CMAKE_CURRENT_LIST_DIR}
       OUTPUT_VARIABLE _tmp )
    string( STRIP ${_tmp} _tmp )
-else ()
-   # Grep first line from file CHANGES if cannot find version from Git
+   # filter invalid descriptions in shallow git clones
+   if (NOT _tmp VERSION_GREATER "0.0.0")
+       set( _tmp "")
+   endif ()
+endif()
+
+# Grep first line from file CHANGES if cannot find version from Git
+if (NOT _tmp)
    file(STRINGS ${CMAKE_CURRENT_LIST_DIR}/CHANGES ALL_VERSIONS REGEX "#")
    list(GET ALL_VERSIONS 0 _tmp)
    string(REPLACE "#" "" _tmp "${_tmp}")

